posts/2017-04-12-serverless-v1.11.0.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Separate Package and Deploy command, CloudWatch Log event support with Serverless v1.11
```

Comment:
@pmuens do you prefer one of the formats David mentioned over the other? We choose one and stick with one and be consistent.